podvm-mkosi/mkosi.profiles/debug.conf

```diff
@@ -0,0 +1,2 @@
+[Content]
+Environment=VARIANT_ID=Debug
```

Comment:
is this capitalized intentionally? `Debug` vs `debug`

Reply:
So if we use profiles instead of the "isDebugEnable"-file workaround that is currently being used, we need to bump the mkosi version, also for the nix flake, right?

Reply:
OK. It should not be capitalized.